apollo-router/src/lib.rs

```diff
@@ -69,6 +69,7 @@ pub mod services;
 mod spec;
 mod state_machine;
 mod test_harness;
+pub mod tracer;
```

Comment:
Would prefer `tracing` as I could see us adding other trace related things in future.

Reply:
I'll change it.